config-overrides/index.js

```diff
@@ -123,7 +124,7 @@ function override(config, env) {
     if (env !== 'development') {
         config.plugins.push(new SSRPlugin('popup.html', src('./src/extension/popup-page/index.tsx')))
         config.plugins.push(new SSRPlugin('index.html', src('./src/index.tsx')))
-        polyfills.map(x => fs.copyFileSync(x, path.join(publicPolyfill, path.basename(x))))
+        polyfills.map(x => void fs.copyFileSync(x, path.join(publicPolyfill, path.basename(x))))
```

Comment:
Why add a `void` here?

Reply:
It indicates that the return value is not actually used. We could also change it to `forEach` loop, if you like.